deploy/ocp/manifests/latest

```diff
@@ -1 +1 @@
-./0.14.1
\ No newline at end of file
+./0.14.2
```

Comment:
can you make sure to make the release in a separate commit?

(or, if you don't want to couple the release process to this PR, can you commit only /deploy/chart and /manifest changes and leave the others out?)

Reply:
we're not overly strict about this (yet) but rule of thumb is that .z bumps are for small patches/fixes. A lot has changed since 0.14.1, so 0.15.0 seems reasonable.

Reply:
In the interest of getting this in quickly I'll just commit the deploy/chart and /manifest changes for now.